docs/pages/blog/v6-beta-pickers.md

```diff
@@ -0,0 +1,135 @@
+
+<video style="margin-bottom:5px; max-width:608px" autoplay muted loop playsinline >
+  <source src="/static/blog/v6-beta-pickers/smart-field.mp4" type="video/mp4" />
+</video>
+
+For a quick comparison, let's check out how the old and new approaches behave in typical scenarios.
+
+<video style="margin-bottom:24px" autoplay muted loop playsinline >
+  <source src="/static/blog/v6-beta-pickers/quick-comparison-fields.mp4" type="video/mp4" />
+</video>
+
+Please, try it out for yourself in the live demo below.
+
+<iframe
+  src="https://codesandbox.io/embed/date-field-demo-pb87v0?fontsize=12&hidenavigation=1&module=%2F"
+  style="width:100%; height:200px; border:0; border-radius: 4px; overflow:hidden;"
+  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
+
+### Fields roadmap
+
+The Fields are still in beta, and there are many improvements on the way. The following list describes some of the most notable features planned.
+
+- **[Smart pasting](https://github.com/mui/mui-x/issues/7253)**
+
+  Date strings pasted in Fields will be parsed and matched with the sections of the date.
+
+- **[Support days of the week](https://github.com/mui/mui-x/issues/7254)**
+
+  Users will be able to select a day in the week just like the can select a month.
+
+We'd love to hear your [feedback](https://github.com/mui/mui-x/issues/new/choose) to help us improve these components even further.
+
+## Other significant improvements
+
+Now that you've seen what's fundamentally changed with the Pickers, let's briefly review some other noteworthy improvements.
+
+- **[Customization of layout and internal components](https://next.mui.com/x/react-date-pickers/custom-layout/)**
+
+  Combining the concept of slots with the grid layout, you now can rearrange, extend, and customize most of the components used internally by the Pickers' views.
+
+- **Default render input**
+
+  You don't need to provide the usual `TextField` on `renderInput` anymore unless you need it customized.
+  In which case you can do it by:
+
+  1. Customizing through props via `field` and `input` slots ([codebox example](https://codesandbox.io/s/customizing-fields-with-props-o66r1c?file=/demo.tsx)).
+  2. Build a new field component using the Field's headless API (documentation coming soon).
+  3. Use an entirely [custom text field](https://next.mui.com/x/react-date-pickers/date-picker/#custom-input-component) to suit your use case.
+
+- **Drag to edit**
+
+  Editing a date range is even easier now with the new drag-and-drop interface. Change `start` and `end` dates at will.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline >
+  <source src="/static/blog/v6-beta-pickers/edit-drag.mp4" type="video/mp4" />
+</video>
+
+- **Range shortcuts**
```

Comment:
@alexfauquette, do you think would be safer to say we'll have it for beta.2? Or is it close enough already of merging? 

```suggestion
- **Range shortcuts** (available from v6.0.0-beta.1)
```

Reply:
I think we can merge it and do follow ups
It would be good enough for a stable release
So for a beta release, the feature is more than good enough !